### PR TITLE
Added minimal Dockerfiles and docker-compose.yaml

### PR DESCRIPTION
**Pull request by njalooo**
This setup builds a docker image according to the build instructions, and nothing more, except for passing the --local header, which makes the service listen to all ip's. This is needed for docker, and access can be controlled via port-forwarding, a proxy, or similar.

I did not test the Nvidia setup, With AMD Radeon it works as intended.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - ./models/vae:/ComfyUI/models/vae
+      - ./models/unet:/ComfyUI/models/unet
+      - ./models/clip:/ComfyUI/models/clip
+      - $OUTPUTDIRECTORY:ComfyUI/output
+    ports:
+      - 8188:8188
+    devices:
+      - "/dev/dri:/dev/dri"
+      ## For AMD GPU
+      - "/dev/kfd:/dev/kfd"
```
Comment (displague) [inline docker-compose.yaml:21]: Are any nvidia devices needed or is it assumed that the nvidia docker binary manages this?

On my machine (1050TI, 560.35.03, CUDA 12.6, Kernel 6.8.0), it looks like this:
```
$ lsof -p $(pidof python) 2>/dev/null | grep /dev | grep -v -e zero -e pts -e shm  | awk '{print $9}' | sort -u 
/dev/nvidia0
/dev/nvidia-caps/nvidia-cap2
/dev/nvidiactl
/dev/nvidia-uvm
```

File: AMD.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.12.5-bookworm
```
Comment (displague) [inline AMD.Dockerfile:1]: Would pytorch serve as a better base?
https://hub.docker.com/r/pytorch/pytorch/tags

File: NVIDIA.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
+FROM python:3.12.5-bookworm
+
+RUN pip install jinja2 numpy
+RUN pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/rocm6.0
+#RUN pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121
```
Comment (displague) [inline NVIDIA.Dockerfile:5]: The nvidia dockerfile would need the CUDA URL

File: docker-compose.yaml (hunk 1 above)
Comment (njalooo) [reply docker-compose.yaml:21]: Nvidia GPU's should be made directly aviable under /dev/dri by the linux kernel, as far as i know.

File: NVIDIA.Dockerfile (hunk 3 above)
Comment (njalooo) [reply NVIDIA.Dockerfile:5]: You are right, my bad. I only changed the commented unstable url

Comment (njalooo) [reply NVIDIA.Dockerfile:5]: Fixed

File: AMD.Dockerfile (hunk 2 above)
Comment (njalooo) [reply AMD.Dockerfile:1]: Probably yes, let me try. It might not require all the build packages after all.

**Comment by rizlas** [comment]
What about #1469 ?

**Comment by njalooo** [comment]
> What about #1469 ?

Ah, i wasn't aware of it. Seems a lot more fleshed out, so i'm going to close this pull request.

**Pull request closed without merging**